tests/zipkin_test.py

```diff
@@ -885,3 +885,464 @@ def test_create_headers_for_new_span_custom_tracer(gen_mock):
     assert expected == zipkin.create_http_headers_for_new_span(
         tracer=tracer,
     )
+
+
+@mock.patch('py_zipkin.zipkin.create_attrs_for_span')
+def test_extract_zipkin_attrs_from_headers_single_sample_only(mock_create_attrs):
+    mock_create_attrs.return_value = 'xyz'  # just a sentinel
```

Comment:
I'd get rid of this and let the function return a real ZipkinAttr.

In the cases where you don't care about the exact value of the ids (like when you only pass the sample flag), just check that specific value.

This will also get rid of the extra `assert mock_create_attrs.mock_calls` making the tests easier to read

Reply:
Done